### PR TITLE
REDCap UI Tweaker version 1.8.0

### PR DESCRIPTION
**Pull request by richdooley**
* Added external modules simplified view.
* Added versionless URLs.
* Added expanded annotation display in the instrument designer.
* Added form display logic to the codebook simplified view.
* Added new alternate status icons for 'unverified' and 'partial survey response'.
* Improvement: Copying from the simplified views in Firefox now retains formatting.

### CODE REVIEW

File: REDCapUITweaker.php [hunk 1]
```diff
@@ -1963,6 +2196,63 @@ function provideUserRoleName()
...
+          else
+          {
+            this.action = vFullURL.replace(/#.*$/,'')
+          }
+        }
+      })
+      var vBaseElem = $('<base>')
+      vBaseElem.attr('href',vFullURL)
+      $('head').append( vBaseElem )
+      var vOldURL = vFullURL.slice( 0, vVersionIndex + 8 + redcap_version.length )
```
Comment (ninaclayton) [inline REDCapUITweaker.php:2229]: I know Nick does not like these hard coded values, can you add as constant?

**Review by ninaclayton — APPROVED** [review]
A few comments but happy to approve.

File: REDCapUITweaker.php (hunk 1 above)
Comment (NicholasHilken) [reply REDCapUITweaker.php:2229]: I generally don't like string literals, but in this case I'm not too worried about replacing 'redcap' with a constant, as this is so baked into the redcap system.

**Review by NicholasHilken — APPROVED** [review]
Thanks Richard.  I took the liberty of resolving a comment that Nina anticipated I might raise!

**Pull request merged**